app/assets/stylesheets/style.scss

```diff
@@ -734,7 +747,7 @@ label:hover {
     height: 60px;
     font-size: 45px;
     display: inline-block;
-    font-family: "Lato";
+    font-family: $font_0;
```

Comment:
Name of variable `font_0` should be written in all lowercase letters with hyphens instead of underscores